### PR TITLE
Enemy ai attacking state

### PR DESCRIPTION
**Pull request by Kenster833**
Alright, here we go again.
This time its based off the adjustments Jeff made and showed in class
Any adjustments let me know
Additions let me know

### CODE REVIEW

File: Source/SteveSloth/Public/MyEnemyAttackingState.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "MyEnemyBaseClass.h"
+#include "MyEnemyBaseState.h"
+#include "MyPlayer.h"
+#include "MyEnemyAttackingState.generated.h"
+
+UCLASS()
```
Comment (CrazyTobi) [inline Source/SteveSloth/Public/MyEnemyAttackingState.h:19]: I would add blueprintable to the UCLASS

**Review by CrazyTobi — CHANGES_REQUESTED** [review]
see comment, aside from that it looks good to me

**Review by CrazyTobi — APPROVED** [review]
Looking good

**Review by AMacLeod99 — APPROVED** [review]

**Review by Saretzky — APPROVED** [review]
looks good

**Review by Jeff-Moreau — APPROVED** [review]
Looks Good

**Pull request merged**